pom.xml

```diff
@@ -65,7 +65,8 @@
         <jose4j.version>0.9.6</jose4j.version>
 
         <!-- Test dependencies -->
-        <strimzi.version>0.43.0</strimzi.version>  <!-- when bumping strimzi, be sure to comment out the kafka.version override in the systemtest module's pom -->
+        <strimzi.version>0.43.0
+        </strimzi.version>  <!-- when bumping strimzi, be sure to comment out the kafka.version override in the systemtest module's pom -->
```

Comment:
This is weird as well. Is this done because of the comment? There are several lines in the main pom like that

Reply:
I'll try and revert the pom changes whey weren't really meant to be part of this

Reply:
This specific instance was because the comment was too long so I've moved it to the line above.